community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/TicketedProcessing.java

```diff
@@ -153,15 +153,24 @@ public void submit( FROM job )
     {
         return future( () ->
         {
-            while ( input.hasNext() )
+            try
```

Comment:
Is this change just a nice find or it has something to do with data buffer size problem?